### ISSUE TITLE
Error: "Send this information to developer : loadProfileType : SLP"

### ISSUE THREAD

**Issue by schmupu** [ioBroker/ioBroker.discovery#65]
Hello,

I get following error since today:

```
discovergy.0 | 2019-09-01 09:13:48.971 | error | Send this information to developer : loadProfileType : SLP
-- | -- | -- | --
discovergy.0 | 2019-09-01 09:13:48.971 | error | Information received from Discovergy which is not yet part of this adapter
discovergy.0 | 2019-09-01 09:13:48.965 | error | Send this information to developer : fullSerialNumber : XXXXXXXXXX
```
I restarted the instance, but this did not help. I can logon at the discovergy homepage without problems. I hope you can help?

**Comment by Apollon77** [comment]
Please re-create in correct adapter :-))

**Comment by HabibA007** [comment]
what?
I set the correct user and password.... in Objects I got valid values...

What you mean with "re-create an correct adapter"? delete them and setup again?
That I've done also....

`

discovergy.0 | 2020-02-15 14:47:31.932 | error | (4978) Information received from Discovergy which is not yet part of this adapter
-- | -- | -- | --
discovergy.0 | 2020-02-15 14:47:31.931 | error | (4978) Send this information to developer : fullSerialNumber : 1ESY116XXXXXXX
discovergy.0 | 2020-02-15 14:47:31.929 | error | (4978) Information received from Discovergy which is not yet part of this adapter
discovergy.0 | 2020-02-15 14:47:31.928 | error | (4978) Send this information to developer : manufacturerId : ESY
discovergy.0 | 2020-02-15 14:47:31.926 | error | (4978) Information received from Discovergy which is not yet part of this adapter

`
